_config.yml

```diff
@@ -5,17 +5,17 @@
 # Replace the baseurl with the name of this repository. For example, if the repository is named
 # "my-site-repository", then update the baseurl like so:
 # baseurl: "/my-site-repository"
-baseurl: "/open-sdg-site-starter"
+baseurl: "/nepal-site"
 # The next line should be the URL of your data repository published site. To find this,
 # go to your data repository in Github, and click "Settings", then scroll down to the
 # "Github Pages" section. Copy/paste the link in the green section.
-remote_data_prefix: "https://open-sdg.org/open-sdg-data-starter"
+remote_data_prefix: "https://open-sdg-nepal.github.io/nepal-data/"
```

Comment:
@jwestw I think `/` needs to be removed from URL

Reply:
Thanks I'll try that